openml_OS/models/api/v1/Api_data.php

```diff
@@ -503,6 +531,11 @@ private function status_update($data_id, $status) {
       return;
     }
     
+    if ($status == 'active' && !$this->user_has_admin_rights) {
```

Comment:
This looks a bit weird. Should a dataset not always be deactivated first, even by the admin? 
If a user wants to remove her dataset, she also has to deactivate it first?

Reply:
this basically states: only admins can activate a dataset.

Reply:
ok.